bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -717,9 +717,17 @@ async fn handle_event_bus_event(
                     p.on_platform_init_start(event.time);
                     drop(p);
                 }
-                TelemetryRecord::PlatformInitReport { metrics, .. } => {
+                TelemetryRecord::PlatformInitReport {
+                    metrics,
+                    initialization_type,
+                    ..
```

Comment:
Would it make sense to use the `InitPhase` (init – invoke) too? 🤔 Although I feel like `InitializationType` is good enough right now